.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: node_js
+node_js:
+  - "6.5.0"
+env:
+  - CXX=g++-4.8
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - g++-4.8
+services:
+  - postgresql
+before_script:
+  - psql -c 'create database ce100_test;' -U postgres
```

Comment:
Yes we should decide on a common name for the database across our plugins but at the moment it's not a blocker and more an enhance feature.